src/dedicated/architecture/scalability.md

```diff
@@ -0,0 +1,16 @@
+# Scalability
+
+Part of the original design goal of Platform.sh’s Triple Redundant Architecture was to ensure scalability in times of load spikes outside of the bounds of the original traffic specs.  Because the cluster is configured as an N+1 architecture, we can respond to legitimate traffic events by removing a node from the cluster, upsizing it, returning it into rotation, and then repeating the process on the next node in turn.
```

Comment:
Maybe we should add a glossary term for N+1 architecture?

Reply:
Probably.  What is it actually defined as? :smile: ?  (I think that terminology may predate me.)